### PR TITLE
Sandbox loader: Check if it's a grafana org plugin

### PR DESCRIPTION
**Pull request by andresmgot**
**What is this feature?**

Extend the check to no try to sandbox plugins with `grafana` as its org.

### CODE REVIEW

File: public/app/features/plugins/sandbox/sandbox_plugin_loader_registry.ts [hunk 1]
```diff
@@ -64,7 +64,11 @@ export async function isPluginFrontendSandboxEligible({
   try {
     // don't run grafana-signed plugins in sandbox
     const pluginMeta = await getPluginSettings(pluginId, { showErrorAlert: false });
-    if (pluginMeta.signatureType === PluginSignatureType.grafana || pluginMeta.signature === 'internal') {
+    if (
+      pluginMeta.signatureType === PluginSignatureType.grafana ||
+      pluginMeta.signature === 'internal' ||
+      pluginId.startsWith('grafana-')
```
Comment (tolzhabayev) [inline public/app/features/plugins/sandbox/sandbox_plugin_loader_registry.ts:70]: curious, do we have cases where signatureType is different for `grafana-` plugins?

**Review by tolzhabayev — APPROVED** [review]

File: public/app/features/plugins/sandbox/sandbox_plugin_loader_registry.ts (hunk 1 above)
Comment (andresmgot) [reply public/app/features/plugins/sandbox/sandbox_plugin_loader_registry.ts:70]: not in the catalog (AFAIK) but it could be unsigned when developing

Comment (academo) [reply public/app/features/plugins/sandbox/sandbox_plugin_loader_registry.ts:70]: I am not convinced with this check. A malicious plugin can potentially escape the sandbox by changing its plugin-id to something starting with `grafana-`

Comment (andresmgot) [reply public/app/features/plugins/sandbox/sandbox_plugin_loader_registry.ts:70]: plugins cannot change their id 😓 so only unsigned plugins would be able to start with `grafana-`. In any case, there is no clear use case for this (only for dev), so happy to not include it.

**Pull request closed without merging**